### PR TITLE
HCAP-816 | HCAP-809 | Removed unfinished TODOs

### PR DESCRIPTION
**Pull request by Zenen-freshworks**
Removed some dead-end code since it wasn't fully implemented

Addresses the following bugs:
https://freshworks.atlassian.net/secure/RapidBoard.jspa?rapidView=206&modal=detail&selectedIssue=HCAP-816
https://freshworks.atlassian.net/secure/RapidBoard.jspa?rapidView=206&modal=detail&selectedIssue=HCAP-809

### CODE REVIEW

File: client/src/pages/private/CohortTable.js [hunk 1]
```diff
@@ -1,10 +1,9 @@
 import React, { useEffect, useState } from 'react';
 import _orderBy from 'lodash/orderBy';
 import Grid from '@material-ui/core/Grid';
-import { Box, Link } from '@material-ui/core';
+import { Box } from '@material-ui/core';
 import { Table } from '../../components/generic';
 import { useToast } from '../../hooks';
```
Comment (fwpushan) [inline client/src/pages/private/CohortTable.js:6]: Please check warning

**Review by fwpushan — CHANGES_REQUESTED** [review]
Please check warning

**Comment by Zenen-freshworks** [comment]
> Please check warning

resolved

**Review by dbayly-freshworks — APPROVED** [review]

**Review by fwpushan — APPROVED** [review]
LGTM

**Pull request merged**